code/modules/mob/living/carbon/life.dm

```diff
@@ -110,9 +108,7 @@
 	if(!breath || (breath.total_moles() == 0) || !lungs)
 		if(reagents.has_reagent("epinephrine") && lungs)
 			return
-		var/oxy_loss = min(losebreath, 1)
-		adjustOxyLoss(oxy_loss)

```

Comment:
This part of the code wasn't actually running at all, they are taking damage from somewhere else somehow